app/components/works/add_files_component.rb

```diff
@@ -3,16 +3,14 @@
 module Works
   # The widget that uploads files to active storage and attaches them to the work.
   class AddFilesComponent < ApplicationComponent
+    delegate :max_upload_files, to: :Settings
```

Comment:
tested similar in rails console on H3, see https://github.com/sul-dlss/hungry-hungry-hippo/pull/745/files#r1962481105